### DISCUSSION TITLE
Modifying attribute points for a metatype

### DISCUSSION THREAD

**Discussion by Aecelot** [chummer5a/chummer5a#5012] [Q&A]
Dear Chummers,

**Context:**
The past week I have been fiddling with chummer XML documents to try and implement a version of the AI-metatype. However, it seems like the value for the attribute points available during chargen is hardcoded in Chummer. 

**Specific:**
Normal A-priority Attributes in the Sum-To-Ten build method have 24 attribute points, and the AI-metatype gets half of that with 12 points. I have found the line of code that applies this halving in the metatypes.xml, which is:

      halveattributepoints /

When I take this out, it correctly resets it to the normal 24 attribute points.

**Question:**
Now for my question, I'd like to implement a modified value (like 16 attribute points), is there any way to do this in an amend_file without going into the source code?

I have tried:

      doubleattributepoints /
      quarterattributepoints /
      attributepoints>16</attributepoints

But none of that seems to work.

Do any of you have any suggestions?

Kind regards,

Aecelot

**Comment by chummer5a** [comment]
There's nothing in place to override that. You'd need to modify SelectMetatypePriority.cs to change the behaviour.

**Reply by Aecelot** [reply]
Alright, thank you for the insight! I'll look into it.
